pkg/syncthing/syncthing.go

```diff
@@ -168,9 +169,9 @@ func New(dev *model.Dev) (*Syncthing, error) {
 		hash = []byte("")
 	}
 
-	rescanInterval := os.Getenv("OKTETO_RESCAN_INTERVAL")
-	if rescanInterval == "" {
-		rescanInterval = "300"
+	compression := "metadata"
+	if dev.Sync.Compression {
+		compression = "always"
```

Comment:
are there other potential options here? if so, maybe dev.Sync.Compression shouldn't be a bool but an enum?

Reply:
There are other compression options, but I would keep it as a bool value for now to enable it or disable it